services/wfm/src/main/java/org/openkilda/wfm/topology/AbstractTopology.java

```diff
@@ -202,27 +205,64 @@ public final T getConfig() {
      *
      * @param topic Kafka topic
      * @return {@link KafkaSpout}
+     * @deprecated should be replaced by {@link AbstractTopology#buildKafkaSpout(String, String)}.
      */
+    @Deprecated
```

Comment:
We need this method, at least for "stats". They need to call `.setFirstPollOffsetStrategy(...)` by their own.

I like builder approach more than serveral set of hardcoded configs.

Reply:
you are right, I'll fix it